Flunky TestValidateParsedMessageAgainstTheParams

ref https://github.com/babylonlabs-io/babylon/actions/runs/11685988374/job/32540724840
```shell
--- FAIL: TestValidateParsedMessageAgainstTheParams (0.08s)
    --- FAIL: TestValidateParsedMessageAgainstTheParams/valid_create_delegation_message (0.00s)
        validate_parsed_message_test.go:837: 
            	Error Trace:	/home/runner/work/babylon/babylon/x/btcstaking/types/validate_parsed_message_test.go:837
            	Error:      	Received unexpected error:
            	            	
            	            	cosmossdk.io/errors.(*Error).Wrapf
            	            		/home/runner/go/pkg/mod/cosmossdk.io/errors@v1.0.1/errors.go:155
            	            	github.com/babylonlabs-io/babylon/x/btcstaking/types.ValidateParsedMessageAgainstTheParams
            	            		/home/runner/work/babylon/babylon/x/btcstaking/types/validate_parsed_message.go:[66](https://github.com/babylonlabs-io/babylon/actions/runs/11685988374/job/32540724840#step:6:67)
            	            	github.com/babylonlabs-io/babylon/x/btcstaking/types_test.TestValidateParsedMessageAgainstTheParams.func25
            	            		/home/runner/work/babylon/babylon/x/btcstaking/types/validate_parsed_message_test.go:826
            	            	staking time 62 is out of bounds. Min: 1000, Max: 100000: the BTC staking tx is not valid
            	Test:       	TestValidateParsedMessageAgainstTheParams/valid_create_delegation_message
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -39,6 +39,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Improvements
 
+* [#252](https://github.com/babylonlabs-io/babylon/pull/252) Fix
+flunky TestValidateParsedMessageAgainstTheParams
 * [#229](https://github.com/babylonlabs-io/babylon/pull/229) Remove babylond
 e2e before upgrade
 * [#250](https://github.com/babylonlabs-io/babylon/pull/250) Add tests for

--- a/x/btcstaking/types/validate_parsed_message_test.go
+++ b/x/btcstaking/types/validate_parsed_message_test.go
@@ -43,7 +43,7 @@ func testStakingParams(
 		MinStakingValueSat:     100000,
 		MaxStakingValueSat:     int64(4 * 10e8),
 		MinStakingTimeBlocks:   1000,
-		MaxStakingTimeBlocks:   100000,
+		MaxStakingTimeBlocks:   10000,
 		SlashingPkScript:       slashingPkScript,
 		MinSlashingTxFeeSat:    1000,
 		MinCommissionRate:      sdkmath.LegacyMustNewDecFromStr("0.01"),

